src/sage/algebras/orlik_solomon.py

```diff
@@ -446,6 +446,33 @@ def degree_on_basis(self, m):
         """
         return len(m)
 
+    def as_cdga(self):
+        """
+        Return the commutative differential graded algebra corresponding to self, with
+        trivial differential.
```

Comment:
```suggestion
        r"""
        Return the commutative differential graded algebra corresponding to ``self``
        with the trivial differential.
```

Reply:
Sounds good.

Reply:
Once this comment is addressed, then it will be a positive review.